tests/api/iotrace_lat_hist_parser.py

```diff
@@ -37,6 +41,78 @@ def __init__(self, hist):
                                hist['total']['range'],
                                hist['total']['unit'])
 
+    @staticmethod
+    def build_histo_from_fio_job(reference_hist, fio_job, io_dir, avg_lat_diff=None):
+        hist_copy = copy.deepcopy(reference_hist)
+        fio_hist = (hist_copy.write if
+                    io_dir is ReadWrite.write else
+                    hist_copy.read)
+
+        for i in fio_hist.items:
+            i.count = 0
+
+        fio_job_bins = (fio_job.write.clat_ns.__dict__['bins'].__dict__ if
+                        io_dir is ReadWrite.write else
+                        fio_job.read.clat_ns.__dict__['bins'].__dict__)
+
+        if avg_lat_diff is not None:
+            fio_normalized_bins = dict()
+            # offset fio samples by 'average' latency
+            for key in fio_job_bins:
+                fio_lat = int(key)
+                fio_lat_offset = fio_lat - avg_lat_diff
+                fio_normalized_bins[fio_lat_offset] = fio_job_bins[key]
+                fio_bins = fio_normalized_bins
+        else:
+            fio_bins = fio_job_bins
+
+        for bin_key in fio_bins:
+            for i, item in enumerate(fio_hist.items):
+                if int(bin_key) >= item.begin and int(bin_key) <= item.end:
+                    fio_hist.items[i].count += fio_bins[bin_key]
+                    break
+
+        return hist_copy
+
+    @staticmethod
+    def calc_histo_correlation(fio_hist, iotracer_hist):
```

Comment:
Suggestion - use correlation calculation from numpy

Reply:
I'd leave it as it is for now, hope that's ok :)